srcpkgs/refind/files/refind-kernel-hook.conf

```diff
@@ -18,5 +18,11 @@ UPDATE_REFIND_CONF=0
 # /efi/EFI/BOOT/refind.conf
 REFIND_CONF=/boot/EFI/refind/refind.conf
 
-# addition kernel cmdline
+# Set the label of the Void Linux filesystem
+REFIND_LABEL="Void Linux"
+
+# Set the path pointing to where rEFInd should look for vmlinuz and initramfs files, e.g. /boot
+REFIND_BOOT_PREFIX=""
```

Comment:
As above, leave the default value commented and perhaps make the documentation more explicit:
```suggestion
# Prefix prepended to kernel and initramfs paths in Void boot entries
#REFIND_BOOT_PREFIX=""
```